p4-16/psa/PSA.mdk

```diff
@@ -146,49 +143,105 @@ of native and non-native operations. Target documentation should highlight such
 implementations, so that application programmers are aware of the
 potentially significant performance penalties.
 
-Semantics of behavior for multiple calls to PRE APIs
+The pseudocode below defines where copies of packets will be made
+after the Ingress control block has completed executing, based upon
+the contents of several metadata fields in the struct
+`psa_ingress_output_metadata_t`.
+
+```
+[INCLUDE=psa.p4:Metadata_ingress_output]
+
+    psa_ingress_output_metadata_t ostd;
 
-The semantics of calling the PRE APIs is equivalent to setting
-intrinsic metadata fields/bits and assuming that the PRE looks up the
-fields in the following order: drop, truncate, multicast, clone,
-output_port.
+    if (ostd.clone) {
+        create a copy of the packet to the clone target
```

Comment:
what is the `egress_port` of the cloned packet? can it be set here or is it assumed to be the same as the original packet?